common/structs/src/configs.rs

```diff
@@ -13,7 +13,7 @@ pub struct SovereignConfig<M: ManagedTypeApi> {
     pub min_validators: u64,
     pub max_validators: u64,
     pub min_stake: BigUint<M>,
-    pub opt_additional_stake_required: Option<ManagedVec<M, StakeArgs<M>>>,
+    pub opt_additional_stake_required: Option<ManagedVec<M, EsdtTokenPayment<M>>>,
```

Comment:
It was ok with `StakeArgs` here because only id + amount should be defined here.